dotCMS/src/integration-test/java/com/dotmarketing/portlets/workflows/actionlet/SaveContentActionletTest.java

```diff
@@ -133,4 +284,53 @@ public void test_Publish_With_Save_Contentlet_Actionlet_Tag () throws DotSecurit
         Assert.assertTrue(contentletPublished.isLive());
     }
 
+    private void checkContentSaved(Contentlet contentletSaved) throws DotDataException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'contentletSaved' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5337055)